pkg/cmd/roachtest/tests/cluster_to_cluster.go

```diff
@@ -244,9 +244,19 @@ type replicateKV struct {
 	// the number of rows inserted into the cluster before c2c begins
 	initRows int
 
-	// max size of raw data written during each insertion
+	// initWithSplitAndScatter splits and scatters the kv table after
+	// initialization. This should be set for tests that init little or no data.
+	initWithSplitAndScatter bool
```

Comment:
whats the thinking behind making this an on and off bool instead of splitting and scattering that scales with the size of the roachtest? I'd assume even a large roachtest would benefit from the data being split and scattered?

Reply:
Good q!

The kv workload will run split and scatter after the insertion step.  After inserting 2 TB of kv table data though, kv has had enough time to split and scatter that table across the cluster. I also found that ALTER TABLE SCATTER on this 2tb table ran for multiple hours before I killed that run of the test. Perhaps I should tell kv 😅 

Initing a couple mb of kv table as other c2c tests do, however, does not lead kv to naturally split and scatter the table by the time the c2c job begins, which will lead the job to create a non-distributed initial dist sql plan because the source side dsp.PartitionSpans may only assign one node all of the data. 

I'll write up a comment that explains this.